app/components/pages/SubmissionWizard/steps/Submission/SubjectAreaDropdown.js

```diff
@@ -142,7 +148,7 @@ class SubjectAreaDropdown extends React.Component {
           ),
         },
         backgroundColor: this.props.theme.backgroundColor,
-        minHeight: this.props.gridUnit,
+        minHeight: `48px`,
```

Comment:
This could use `minHeight: ${th('space.5')}`  instead so it uses the grid sizing still